tests/functional/test_tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = py26,py27,pypy,py32,py33,py34,pypy3
```

Comment:
Hmmm, they have the same other as in the main tox.ini (by python version). Should I reorder?